CHANGES.md

```diff
@@ -1,2 +1,11 @@
+Version 2.1:
+ - Change default Docker version to 1.11.2, was 1.5.0
```

Comment:
I'll try one more time to pitch to take out the default.  I don't want this to 'just work', or at least I don't care if it does or not.  To me, this is the wrong place to decide what version to use.  Forcing users of this code to decide what version to use is a good thing to me.  It puts the decision in the right place, and it makes it easy to change.  Putting the information here makes it hard to change.